client/payment-details/summary/test/__snapshots__/index.tsx.snap

```diff
@@ -698,37 +754,11 @@ exports[`PaymentDetailsSummary renders capture section correctly 1`] = `
         </li>
       </ul>
     </div>
-    <div
```

Comment:
Is the removal of the capture section expected? 🤔

Reply:
Maybe due to the snapshot changes, as explained in the earlier note. ( static date changed to relative time )